packages/donor/src/components/AppHeader/AppHeader.tsx

```diff
@@ -78,6 +93,25 @@ export default function ButtonAppBar({
           </Grid>
         </Grid>
       </Toolbar>
+      <div>
+        <SideDrawer
+          isOpen={showSideBar}
+          close={() => setShowSideBar(false)}
+          navProps={{
+            navItemsProps: [
+              {
+                title: "החשבון שלי",
+                onClick: () => history.push("/" + MainNavigationKeys.MyProfile),
+              },
+              {
+                title: "התנתק",
+                onClick: () => firebase.auth().signOut(),
```

Comment:
1. Take signOut() as a prop here
2. Remove the sign out option from the profile page

Reply:
I started with the signout coming from outside but then you have to specify it and it's not always used to it becomes a little clumsy.
Is there a specific reason (testing?) why it's problematic to have it inlined here?

Reply:
2 = done